### ISSUE TITLE
bi-width fonts

### ISSUE THREAD

**Issue by hackerb9** [uobikiemukot/yaft#48]
![w3m sony.jp](https://user-images.githubusercontent.com/10174223/64958798-bbb3a880-d844-11e9-97fa-7cb3599e9ce9.png)

The README states that multiple fonts can be used, but they must all be the same size. What's not mentioned is that if one uses a single font that is multiple sizes, `yaft` works correctly. [See example.]

![biwidthfontexample](https://user-images.githubusercontent.com/10174223/64957739-6bd3e200-d842-11e9-8823-fb15820184b9.png)

Here you can see I am editing a text file that has characters of different widths. I am using a bi-width font I downloaded from http://openlab.ring.gr.jp/efont/unicode/ called "b24.bdf".  [b24.bdf.gz](https://github.com/uobikiemukot/yaft/files/3616217/b24.bdf.gz)

I suggest the README perhaps should mention the possibility of a using a bi-width font.

Also, it would be nice if `mkfont_bdf` would correctly merge two fonts (one narrow and one wide) into glyph.h. If that's not possible, please consider implementing an option like `xterm` has to specify a second font just for wide font characters. 

Thank you.

**Comment by uobikiemukot** [comment]
Yes, `mkfont_bdf` allows bi-width font (requires that narrow font has just half width of wide font).
Usually Japanese fonts is a bi-width font, So I forgot to mention it in README.
I'll update README.
